javascript/javascript1/week4/colorValuesApp.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <script>
+         [["white", "goodness"], ...]
+​
+function colourAssociation(array) {
+  let newArray = []
+  for (let i = 0; i < array.length; i++) {
+    const key = array[i][0]
+    const value = array[i][1]
+​
+    const colorPair = {[key]: value}
+    newArray.push(colorPair)
+  }
+​
+  return newArray
+}
+
+console.log(colourAssociation[blue:aqua,green:beauty]);
```

Comment:
You've forgotten the parentheses to invoke your function here!  

```suggestion
console.log(colourAssociation([blue:aqua, green:beauty]));
```